### ISSUE TITLE
Getting ping timeout for no reason

### ISSUE THREAD

**Issue by DaHooni** [martynsmith/node-irc#468]
As the title says I have a few accounts im trying to connect to an irc server, however somehow it gives me ping timeout, but just in one of my pcs, the other runs perfectly.
It isnt my internet connection because I tested it via cable, and both use the same cable to connect.
I wonder if it may be related to memory because the pc that works have 8gb, the one that doesnt have 4gb.
This is what I get when its trying to connect:
![error](https://cloud.githubusercontent.com/assets/19698073/15728573/b1bbfb8a-2833-11e6-965a-bb123f6e77f3.jpg)

And after that I get ping timeout for every single account. I tried with 5 accounts and it isnt connecting either, so its not a server based limitation, cause on my other pc I use like 10 just fine.

**Comment by moshmage** [comment]
are the 5 bots connecting "at the same time" ? as in, can it be that the server might be protecting itself from a connection flood?

**Comment by DaHooni** [comment]
Yes, they do connect all at once. But what I dont understand is, on my other pc they connect like 10 accounts all at once, and they connect just fine, I never got this error. But on the pc that doesnt work I tried even with 3 accounts and I get the same error. So how the flood protection only applies to one of my pc?

**Comment by moshmage** [comment]
In that case, there's something wrong with either the installation of node, or something. As in: IF the code works on one machine but not the other, usually means some dependencies are missing. Check if you can connect using a normal irc-client (on the faulty machine) before starting going deep into the node-irc bug-solving.
